ddsketch/store/store.go

```diff
@@ -20,5 +20,5 @@ type Store interface {
 	KeyAtRank(rank float64) int
 	MergeWith(store Store)
 	ToProto() *sketchpb.Store
-	FromProto(pb *sketchpb.Store)
+	FromProto(pb *sketchpb.Store) Store
```

Comment:
Maybe a comment to highlight that this creates a new store (as opposed to filling the this one).